MetaMask/metamask-extension
# updated CollectibleDetail Storybook Coverage

Fixes #17157 

### Before
<img width="1311" alt="Screenshot 2023-01-23 at 9 56 46 PM" src="https://user-images.githubusercontent.com/39872794/214093757-83126cd6-1dc8-4daa-9cd0-2c14a218226b.png">


### After

https://user-images.githubusercontent.com/39872794/215050964-af1affb6-ee16-49af-9890-2d74ff0f002a.mov


## Manual Testing Steps

Storybook
- Go to the latest build of the storybook in this PR
- Search for CollectibleDetail  in App
- Check the stories, controls, and documentation

## Pre-Merge Checklist


### + If there are functional changes:
